src/components/molecules/MessageList.vue

```diff
@@ -21,6 +21,7 @@
 <script>
 import MessageCard from "./MessageCard.vue";
 import MessageHeader from "../atoms/MessageHeader.vue";
+import EmailService from "../../services/emails/emails.js";
```

Comment:
The Service should only be imported inside Vuex (our modules).

Reply:
Oops, I had put it in there for testing haha